docs/src/pages/guides/interoperability/interoperability.md

```diff
@@ -227,7 +266,8 @@ export default function GlobalCssSlider() {
 ### Change the default styled engine
 
 By default, Material-UI components come with emotion as their style engine. If,
```

Comment:
```suggestion
By default, Material-UI components come with emotion as its style engine. If,
```

Reply:
"components" it's plural I think their is correct.

Reply:
Quite right. Must have read it too quickly!